docs/concepts.md

```diff
@@ -85,9 +91,14 @@ the workspace repository.
 
 A cluster spec contains desired settings when creating a new cluster.
 They currently include:
+
 - API Provider: the cluster orchestration technology. Supported values are `CAPI` (Cluster API) and `xplane` (Crossplane)
 - Cloud Provider: the infrastructure cloud provider. The currently supported values is `aws`, with `gcp` and `azure` support coming later.
 - Type: the cluster type. Some API providers support more than one type. On `aws` cloud, Cluster API supports `kubeadm` and `eks`, whereas Crossplane only supports `eks`.
 - The (worker) node instance type
 - The initial (worker) node count
 - The Kubernetes version
+
+## Base cluster
+
+To know more about basecluster (Arlon gen2 clusters), read it [here](https://github.com/arlonproj/arlon/blob/main/docs/baseclusters.md)
```

Comment:
Can we link to the baseclusters page instead of going to the repository. I think the content is identical.

Reply:
Done